editor/modules/map/editor.ts

```diff
@@ -74,8 +73,9 @@ export class MapEditor implements AfterViewInit, OnDestroy {
     }
 
     commit(): void {
-        let map = this.map_manager.currentMap();
-        if (map) {
+        let _map = this.map_manager.currentMap();
```

Comment:
For this one, the bug is caused by the TypeScript compiler. It was complaining that map could be null within the `if(map) { ... }`. Will more likely be fixed in the next minor upgrade of TypeScript.